File: PT_Boldestio_Python/models/day.py
from dataclasses import dataclass, field
from typing import List
from .place import Place

@dataclass
class Day:
    day: str
    title_es: str
    htmlDescription_es: str
    htmlContent_es: str
    resourceIDs: List[Place] = field(default_factory=list)

    def add_place(self, place: Place):
        # Añade una instancia de Place a la lista resourceIDs
        if not isinstance(place, Place):
            raise TypeError("Expected an instance of Place")
        self.resourceIDs.append(place)

File: PT_Boldestio_Java/src/main/java/script/Day.java
package script;

import java.util.LinkedList;

public class Day {
    private String day;
    private String title_es;
    private String htmlDescription_es;
    private String htmlContent_es;
    private LinkedList<Place> resourceIDs;

    public Day(String day, String title, String htmlDescription, String htmlContent) {
        this.day = day;
        this.title_es = title;
        this.htmlDescription_es = htmlDescription;
        this.htmlContent_es = htmlContent;
        this.resourceIDs = new LinkedList<Place>();
    }

    public void annadirPlace(Place place){
        this.resourceIDs.add(place);
    }
}
